### ISSUE TITLE
[华农兄弟] 华农兄弟看一下兄弟家的香菇顺便摘点粽叶再挖一个竹笋 - https://youtu.be/PWGHxna1Udw

### ISSUE THREAD

**Issue by immoonancient** [immoonancient/YTSubtitles#692]
https://youtu.be/PWGHxna1Udw
<p>&#35270;&#39057;&#21457;&#24067;&#20110; February 19, 2020 at 11:43AM</p>
<p><small><i>&#30001; <a href="https://ifttt.com/applets/104837875d">IFTTT</a> &#33258;&#21160;&#21457;&#24067;</i></small></p>

**Comment by vincenthychow** [comment]
bot, please upload
#https://youtu.be/PWGHxna1Udw

#华农兄弟：看一下兄弟家的香菇，顺便摘点粽叶，再挖一个竹笋
#Huanong Brothers: Let's have a look at our buddy's shiitake mushrooms and take some palm leaves, and also a bamboo shoot along the way.

0:00:00.420,0:00:02.400
# 大家好  今天天氣比較好哦
Hi everyone! The weather today is quite good.

0:00:02.400,0:00:04.440
# 我上山看一下兄弟家的香菇
I'm going onto the hill and have a look at my buddy's shiitake mushrooms.

0:00:04.580,0:00:06.220
# 他都摘了幾千斤啦
He has already picked thousands of kilograms.

0:00:06.500,0:00:07.600
# 就在前面一點
It's just a bit in front of us over there.

0:00:07.920,0:00:09.400
# 這是他家的牛
These are his cattle,

0:00:10.320,0:00:11.240
# 他家的牛
his cattle.

0:00:12.380,0:00:13.000
# 好多哦
A lot!

0:00:16.120,0:00:17.560
# 哦喲  一條小牛哦
Oyo! A calf!

0:00:18.120,0:00:18.980
# 好小的
It's really small.

0:00:21.640,0:00:23.580
# 你看那個小牛  好可愛喲
Look at that calf. It's so cute,

0:00:23.780,0:00:24.460
# 好小
and so small.

0:00:25.040,0:00:26.600
# 今天(註︰原字幕出錯，他說「今年」)我要養一頭牛來
I'll also be keeping one this year.

0:00:28.240,0:00:29.320
# 養到來騎著
So that I can ride,

0:00:29.440,0:00:30.460
# 可以騎牛的
ride a cow.

0:00:32.420,0:00:34.500
# 兄弟家養了差不多30多條牛
My buddy has more than 30 cattle.

0:00:35.260,0:00:37.860
# 今年又下崽啦  可能40條都上去啦
And they gave birth again this year. Perhaps he has more than 40.

0:00:39.760,0:00:42.120
# 我不檔着你的路 從這里走哇
I'm not blocking your way. You can go on this side.

0:00:42.700,0:00:43.580
# 走~
C'mon, go~

0:00:44.420,0:00:45.280
# 不敢過來
They dare not.

0:00:51.440,0:00:52.760
# 我走我的陽光[SIC]道
We depart to our own paths,

0:00:53.620,0:00:54.900
# 你過你的獨木橋
and shall never meet again.

0:01:06.120,0:01:08.660
# 你看這個樹  全部兄弟摘完啦
Look at the trees! My buddy has already picked all.

0:01:09.020,0:01:11.080
# 我們來遲啦  上面還有
We are too late. There are more up there.

0:01:11.420,0:01:12.520
# 要上上面去
Let's go up there.

0:01:26.140,0:01:28.760
# 你看這個好像是銀綫蓮  還是金綫蓮
Look! This one looks like a "silver clematis" or a "golden clematis".

0:01:29.200,0:01:30.160
# 摘不清楚
No ideas.

0:01:30.660,0:01:31.260
# 這個
It's a...

0:01:32.200,0:01:33.640
# 它長在這裡  太少啦
It's growing here. It's too few.

0:01:34.040,0:01:35.920
# 不然拿回去煲雞湯哦
I would take it back to make, make a chicken soup if there are more.

0:01:47.000,0:01:49.840
# 你看  這個有點像紅蘑菇哦  (註︰這都是他的土話吧)
Look! This one looks a bit like red mushroom.

0:01:50.160,0:01:53.420
# 其實它不是的哦 這個是  我也不知道是甚麼菇
But in fact it's not. This is, well, I've no ideas what kind of mushroom it is.

0:01:53.520,0:01:54.320
# 毒蘑菇
It's poisonous,

0:01:54.660,0:01:55.900
# 這個不能吃的哦
not edible.

0:01:58.940,0:02:00.160
# 拍散  打死 打死 打死
Smash it! Kill it, kill it, kill it!

0:02:02.400,0:02:03.280
# 從這上去
Let's go up from this way.

0:02:14.160,0:02:16.920
# 你看  這還有一個小香菇 就看到一個
Look! Here is still a small mushroom. I can see just one.

0:02:19.520,0:02:21.920
# 都來遲啦  都被他們摘掉啦
We're too late. They've already picked everything.

0:02:22.620,0:02:24.520
# 一毎  二[SIC]個  三個
One, two, three

0:02:24.600,0:02:27.100
# 一棵樹上長了十幾個  全部被他們摘光啦
There were more than a dozen on a tree. But they've picked all.

0:02:28.400,0:02:30.660
# 就知道兄弟騙我  說好多  好多
I knew that my buddy faked me. He said he had a lot and a lot.

0:02:31.040,0:02:32.640
# 看他的眼神  我就看得出來
I can tell the truth my simply looking into his eyes.

0:02:33.400,0:02:34.900
# 你看  就摘了一個
Look! I have just one here.

0:02:36.120,0:02:37.200
# 一個  我也帶回去
Doesn't matter. I'm taking it back home.

0:02:38.640,0:02:40.080
# 幸好我帶了一把鋤頭
Fortunately, I'm coming with a hoe.

0:02:40.280,0:02:41.260
# 去挖竹笋去
We can go to get some bamboo shoots.

0:02:42.040,0:02:43.140
# 刀我也帶了
I have a knife too.

0:02:44.440,0:02:45.500
# 去  挖竹笋去
Let's go! It's time for the bamboo shoots.

0:02:51.160,0:02:53.460
# 怎麼兩個  兩個都摘不到
Why can't I have two, just two?

0:02:54.280,0:02:56.260
# 摘兩個多好哇  一個[SIC]一個
With two, each of us could have one. Isn't it great?

0:02:56.920,0:02:58.020
# 今天晚上
We could have a soup,

0:02:59.140,0:03:01.980
# 煮一個  雞蛋香菇湯
with eggs and mushrooms tonight.

0:03:04.440,0:03:05.860
# 第二個都找不出來啦
But this second mushroom is nowhere to be found.

0:03:07.020,0:03:07.980
# 兄弟家
My buddy's,

0:03:08.500,0:03:10.660
# 兄弟家香菇樹都摳哇
my buddy's mushroom trees are becoming stringy.

0:03:11.820,0:03:12.940
# 都說兄弟摳
I know that my buddy has always been stingy.

0:03:14.320,0:03:16.220
# 今天才發現  真的很摳
But I didn't know to what extend, until today.

0:03:19.420,0:03:21.400
# 看到  說我要來啦
He saw, I said that I'm coming,

0:03:21.680,0:03:23.860
# 應該是他昨天就把它全部摘光啦
then he must have picked everything yesterday.

0:03:24.820,0:03:27.540
# 摘光就算了  還摘得一個不剩
Having picked everything is one issue. But not leaving even a single one behind is another issue.

0:03:29.360,0:03:30.540
# 真的一個不剩
No kidding, not even one is left.

0:03:30.720,0:03:31.880
# 第二個都找不出來
There's nothing called a second one.

0:03:38.460,0:03:39.840
# 你看  這裡有一個
Look! Here's one.

0:03:40.180,0:03:41.560
# 肯定是今天長出來的
It must be newly grown today.

0:03:43.660,0:03:44.500
# 兩個
So two

0:03:45.640,0:03:46.780
# 搞個香菇湯
We can make a soup.

0:03:47.840,0:03:49.000
# 應該可以
Why not?

0:03:50.440,0:03:52.040
# 走啦  去挖竹笋去
Let's go! It's time for the bamboo shoots.

0:03:52.940,0:03:55.280
# 兄弟太子厚道啦
My buddy is too, too mean.

0:04:01.740,0:04:02.560
# 小牛
Cowy!

0:04:03.300,0:04:04.040
# 小牛
Cowy!

0:04:05.740,0:04:06.720
# 小牛~
Cowie~

0:04:08.260,0:04:09.040
# 小牛
Cowy!

0:04:10.100,0:04:11.900
# 這條是公的
This is male.

0:04:14.780,0:04:16.080
# 吃得好飽哇
It has its stomach stuffed.

0:04:17.320,0:04:18.620
# 你身上好多[那]個(肉)
You're covered with, you know.. (beef)

0:04:22.600,0:04:23.720
# 摸一下你的肉
Let me feel your beef.

0:04:26.880,0:04:28.400
# 我懷疑你想把我拱到下面去
I suspect that you want to push me down there.

0:04:34.880,0:04:37.480
# 我看這個粽葉 又開始長出嫩的出來啦 (註︰以前的「粽」和「棕」應該是錯誤的混淆了)
Look at these palm leaves. The new ones are coming out again.

0:04:37.880,0:04:38.680
# 我摘幾片來
Let me get a few,

0:04:39.200,0:04:40.660
# 做幾個粽子吃一下
to make a few rice dumplings.

0:04:54.580,0:04:56.020
# 你看  這個葉子
Look! The leaves,

0:04:56.520,0:04:57.640
# 不是很新鮮
are not really that fresh.

0:04:57.840,0:04:59.240
# 不是剛長出來的
They're not newly grown,

0:04:59.400,0:05:00.560
# 去年的老葉子
but are the old one from last years.

0:05:01.780,0:05:03.820
# 但是沒辦法  我想提前吃粽子
Well, anyway. I want to have rice dumplings, now, well in advance.

0:05:05.080,0:05:06.420
# 所以就摘吧
So just pick some.

0:05:17.580,0:05:19.360
# 這個嫩的
This is young.

0:05:21.120,0:05:22.440
# 摘這個嫩一點
I'll take this one, a younger one.

0:05:23.400,0:05:24.220
# 摘幾片
Let me pick a few more,

0:05:24.480,0:05:26.000
# 回去裹幾個粽子
to make some rice dumplings at home.

0:05:27.460,0:05:29.000
# 裏幾個粽子吃一下就好啦
A few rice dumplings would be enough.

0:05:35.940,0:05:36.660
# 差不多
Should be enough.

0:05:36.880,0:05:37.600
# 一個粽子
One rice dumpling

0:05:37.760,0:05:38.540
# 二[SIC]個粽子
Two rice dumplings

0:05:39.220,0:05:40.100
# 三個粽子
Three rice dumplings

0:05:41.580,0:05:42.740
# 四個粽子
Four rice dumplings

0:05:43.220,0:05:44.420
# 五個六個
Five, six

0:05:44.740,0:05:46.420
# 能包  能裏6個粽子
I can make, I can make six rice dumplings.

0:05:46.500,0:05:47.000
# 夠了
It's enough.

0:05:48.300,0:05:49.420
# 我就摘十二個
So I have twelve here.

0:05:50.120,0:05:51.420
# 六個粽子  一個三個
Six rice dumplings, so three each.

0:05:52.820,0:05:55.280
# 要不我四個  你兩個
Or I have four, you have two.

0:05:55.560,0:05:56.360
# 也可以啊
It's also fine.

0:05:56.360,0:05:57.340
# 這個粽子不好吃
The rice dumplings won't taste good.

0:05:57.580,0:05:58.600
# 要到春天的時候
Not until spring...

0:05:58.840,0:06:01.920
# 不是春天  要到這個粽葉剛出來的時候才好吃
No, not spring. I mean it won't taste good until the new palm leaves grow.

0:06:03.140,0:06:04.180
# 摘多幾片
Let's have a few more.

0:06:04.780,0:06:05.860
# 夠啦 不要啦
Enough! No more

0:06:10.200,0:06:11.860
# 不要啦  摘幾片就不要啦
No more. Just a few more, that's it.

0:06:15.500,0:06:16.740
# 啊  這邊有嫩的
Ah! Here are some younger ones!

0:06:17.100,0:06:18.340
# 有嫩的就要多摘一點
I'll take a few more as they're young.

0:06:35.680,0:06:37.060
# 好  不要啦  不要啦
Alright! No more, no more

0:06:37.160,0:06:38.060
# 那邊[SIC]一點[點]就夠啦
A tiny little bit here is enough.

0:06:50.140,0:06:51.360
# 嗯好  夠啦  夠啦
Um! Alright! Enough, enough

0:06:52.860,0:06:55.940
# 你看  夠啦  回去洗乾淨 裏幾個粽子吃一下
Look! It's enough. I'll wash them at home and make a some rice dumplings.

0:06:57.500,0:06:58.620
# 提前吃你們
I'll eat you guys in advance.

0:07:00.340,0:07:01.080
# 回去啦
Let's go back!

0:07:26.260,0:07:27.440
# 可以 摘根繩子
Alright! Get a piece of string,

0:07:28.320,0:07:29.280
# [綁起來] (沒有原字幕)
to tie them up

0:07:29.600,0:07:30.720
# 好  摘定
Okay! Done!

0:07:51.200,0:07:52.360
# 哎呀  有一個
Aya! Here's one!

0:07:53.600,0:07:54.260
# 看到沒
See?

0:07:55.880,0:07:56.520
# 應該
It should be,

0:07:57.180,0:07:59.600
# 應該  大的哦  這個
it should be large, this one.

0:08:01.900,0:08:03.260
# 它爹都那麼大
Its dad is that big,

0:08:03.400,0:08:04.340
# [它肯定很大] (沒有原字幕)
so it must be very big.

0:08:17.120,0:08:19.060
# 可以吧  這個  夠大吧
This is good, isn't it? It's big enough.

0:08:41.780,0:08:43.840
# 就是一個獨苗 沒有啦
It's a single child, no more.

0:08:43.980,0:08:48.160
# 那麼大的竹子  那麼大的竹就長了一個出來
It's such a huge bamboo, such a huge bamboo but it has only one shoot.

0:08:49.500,0:08:51.880
# 你看  今天  收獲滿滿哦
Look! Today, is a fruitful day.

0:08:52.020,0:08:54.980
# 粽葉  竹笋  還有香菇
Palm leaves, bamboo shoot, and mushrooms.

0:08:57.080,0:09:00.040
# 可以  今天的視頻就到這裡啦  回去啦
Nice! The video ends here. Let's go back.

0:09:01.120,0:09:01.900
# 刀哦
My knife

0:09:05.680,0:09:06.720
# 明天我還來
I'll come again tomorrow.

**Comment by vincenthychow** [comment]
已發佈
